functests/tests.sh

```diff
@@ -28,22 +28,22 @@ function test_build() {
     deploy_minio
     uninstall_minio
     uninstall_directcsi
-    # Check uninstall succeeds even if direct-csi is completely gone.
+    # Check uninstall succeeds even if directpv is completely gone.
     "${DIRECT_CSI_CLIENT}" uninstall --crd --force
 }
 
 function do_upgrade_test() {
     wget --quiet --output-document="kubectl-direct_csi_$1" "https://github.com/minio/directpv/releases/download/v$1/kubectl-direct_csi_$1_linux_amd64"
     chmod a+x "kubectl-direct_csi_$1"
 
-    # unmount all direct-csi mounts of previous installation if any.
-    mount | awk '/direct-csi/ {print $3}' | xargs sudo umount -fl
+    # unmount all directpv mounts of previous installation if any.
+    mount | awk '/directpv/ {print $3}' | xargs sudo umount -fl
 
     DIRECT_CSI_CLIENT="./kubectl-direct_csi_$1"
     DIRECT_CSI_VERSION="v$1"
-    image="direct-csi:${DIRECT_CSI_VERSION}"
+    image="directpv:${DIRECT_CSI_VERSION}"
     if [ -n "${RHEL7_TEST}" ]; then
-        image="direct-csi:${DIRECT_CSI_VERSION}-rhel7"
+        image="directpv:${DIRECT_CSI_VERSION}-rhel7"
```

Comment:
the new tag name `directpv:<version>` doesn't apply for older versions. (v1.3.6, v1.4.6 - used for upgrade tests)